downstream/modules/platform/con-controller-control-job-run.adoc

```diff
@@ -0,0 +1,43 @@
+[id="controller-control-job-run"]
+
+= Control where a job runs
+
+If you associate instance groups with a job template, inventory, or organization, a job run from that job template is not eligible for the default behavior. 
+This means that if all of the instances inside of the instance groups associated with these three resources are out of capacity, the job remains in the pending state until capacity becomes available.
+
+The order of preference in determining which instance group to submit the job to is as follows:
+
+. Job template
+. Inventory
+. Organization (by way of project)
+
+If you associate instance groups with the job template, and all of these are at capacity, then the job is submitted to instance groups specified on the inventory, and then the organization. 
+Jobs must execute in those groups in preferential order as resources are available.
+
+You can still associate the global `default` group with a resource, like any of the custom instance groups defined in the playbook. 
+You can use this to specify a preferred instance group on the job template or inventory, but still enable the job to be submitted to any instance if those are out of capacity.
+
+.Examples
+
+* If you associate `group_a` with a job template and also associate the `default` group with its inventory, you enable the `default` group to be used as a fallback in case `group_a` gets out of capacity.
+* In addition, it is possible to not associate an instance group with one resource but designate another resource as the fallback. 
+For example, not associating an instance group with a job template and have it fall back to the inventory or the organization's instance group.
+
+This presents the following two examples:
+
+. Associating instance groups with an inventory (omitting assigning the job template to an instance group) enables you to ensure that any playbook run against a specific inventory runs only on the group associated with it. 
```

Comment:
Perhaps consider replacing "enables you to ensure" with "ensures" so it's more concise?